tests/Bundle/Helper/ApplicationTester.php

```diff
@@ -0,0 +1,130 @@
+<?php
+namespace tests\Clearcode\CommandBusConsole\Bundle\Helper;
+
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Helper\QuestionHelper;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\StringInput;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Output\StreamOutput;
+
+/**
+ * Copied and modified from https://github.com/phpspec/phpspec/blob/master/features/bootstrap/Console/ApplicationTester.php.
+ */
+class ApplicationTester
```

Comment:
This is copy of https://github.com/matthiasnoback/symfony-console-form/blob/master/test/Helper/ApplicationTester.php which is copy of https://github.com/phpspec/phpspec/blob/master/features/bootstrap/Console/ApplicationTester.php ;)